app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,61 @@
+class ConversationsController < ApplicationController
+  before_filter :authenticate_user!
+  helper_method :mailbox, :conversation
+
+  def create
+    recipient_emails = conversation_params(:recipients).split(',')
+    recipients = User.where(email: recipient_emails).all
+
+    conversation = current_user.
+      send_message(
+      recipients, *conversation_params(:body, :subject)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.